### PR TITLE
[TW-469] Update docs for new settings UI

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
Please make sure to pull the branch `tw-469-settings-redesign` when reviewing this PR. There is an older `tw-469-settings-updates` branch which is no longer valid.

### CODE REVIEW

File: src/pages/docs/getting-started/settings.md [hunk 1]
```diff
@@ -82,32 +81,35 @@ When you send a form-data or binary file with a request body, Postman saves a pa
...
+* **Default documentation editor** - Select the default editor you want to use for [editing documentation descriptions](/docs/publishing-your-api/authoring-your-documentation/) in Postman (**Postman editor** or **Markdown editor**).
 
 ### Editor settings
 
 **Editor** settings affect code-related text such as request and response bodies, pre-request scripts, and tests. To revert back to default text settings, select **Reset**.
 
-* **Font Family** - Enter one or more font family names separated by commas. Postman uses the first available font family to display code text.
-* **Font Size (`px`)** - Enter the font size in pixels to use for code text.
+* **Font family** - Enter one or more font family names separated by commas. Postman uses the first available font family to display code text.
+* **Font Size** - Enter the font size in pixels to use for code text.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/getting-started/settings.md:96]: I checked on beta in the `default` track (not sure if this has the latest UI). But I'm seeing "Font size" in sentence case, not title case. Is this correct?

File: src/pages/docs/getting-started/settings.md [hunk 2]
```diff
@@ -82,32 +81,35 @@ When you send a form-data or binary file with a request body, Postman saves a pa
...
 * **Indentation count** - Enter the number of indentation characters to use for each code level.
 * **Indentation type** - Select the indentation character type to use (**Space** or **Tab**).
 * **Auto close brackets** - Turn this on to automatically add a closing bracket when you enter an opening bracket.
 * **Auto close quotes** - Turn this on to automatically add a closing quotation mark when you enter an opening quotation mark.
 
 ### Application
 
+* **Autosave** - Turn this on to automatically save any changes you make to requests and collections.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/getting-started/settings.md:104]: Should we be documenting the "Autosave" option? I have an open ticket about this feature (`TW-1110`), and I think this feature is only available to free and single user teams at the moment (other than the `postman` team).

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM! When I last checked, the Autosave feature isn't GA yet, so we might want to look at that again. Thanks!

File: src/pages/docs/getting-started/settings.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/settings.md:96]: You're right, thanks for seeing that! 🦅 👀

File: src/pages/docs/getting-started/settings.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/settings.md:104]: Thanks for the heads up on this! I do see Autosave on my free account, but it's labeled "beta". So I think it makes sense to wait to document this setting along with TW-1110. For now, I'll comment this out.

Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/settings.md:104]: Sounds good! Thanks!

**Review by amrindersandhu — APPROVED** [review]
Reviewed the UI in screenshots. Good to go!

File: src/pages/docs/sending-requests/troubleshooting-api-requests.md [hunk 3]
```diff
@@ -91,11 +91,11 @@ Issue | Resolving the issue
...
+**SSL certificates** | You may experience issues using HTTPS connections. You can turn off **SSL certificate verification** in [Settings](/docs/getting-started/settings/) by selecting the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> **> Settings > General**. If that doesn't help, your server might be using a client-side SSL connection, which you can configure by selecting the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> > **Settings > Certificates**. Use the [Postman Console](#debugging-in-the-console) to ensure that the correct SSL certificate is being sent to the server. Learn more about [working with certificates](http://localhost:8000/docs/sending-requests/certificates/).
```
Comment (claireriley) [inline src/pages/docs/sending-requests/troubleshooting-api-requests.md:94]:
```suggestion
**SSL certificates** | You may experience issues using HTTPS connections. You can turn off **SSL certificate verification** in [Settings](/docs/getting-started/settings/) by selecting the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> **> Settings > General**. If that doesn't help, your server might be using a client-side SSL connection, which you can configure by selecting the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> > **Settings > Certificates**. Use the [Postman Console](#debugging-in-the-console) to ensure that the correct SSL certificate is being sent to the server. Learn more about [working with certificates](/docs/sending-requests/certificates/).
```

**Review by claireriley — APPROVED** [review]
LGTM! 🌟 noted one link to fix

File: src/pages/docs/sending-requests/troubleshooting-api-requests.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/troubleshooting-api-requests.md:94]: 🤦🏻 Oof, thanks for spotting that Claire!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**